### ISSUE TITLE
archive of past events content?

### ISSUE THREAD

**Issue by SaraMahallati** [UofTCoders/council#216]
I think we should put up the lesson content of each session somewhere on our website so that it's available to the people who has attended to go back and review the same code and examples that was taught in the session.  Especially since we are repeating the intro sessions every term, but the lesson content is the one that was taught one time (probably the first time). Although it is still useful to go through the instructions, but it is much more efficient  as a beginner if you review the exact lesson content (examples, codes, etc.) that you once heard in the class. 

e.g. the recent Intro session to R content is totally different from the one that we have in our lesson content webpage. There were a few people that fell behind in typing along with Luke and continued by just observing (or taking notes), and asked if the lesson will be available to them to review on their own time. I think they will be quite confused as a beginner if I refer them to the current lesson content.

Can we have a webpage for the archive of past events? or keep the past events in the events page to have a link to their lesson content? (I know the page will get too long after a while if we want to keep all events), any suggestions?

**Comment by lwjohnst86** [comment]
Yes, good idea! It shouldn't be too hard to implement too. I could take a stab at it later in the month/next month. I think I'll change my position to be "technical admin, remote" or something like that :) I have a couple ideas on how to streamline what we do, but just hadn't had time to try developing them.
